src/app/page.tsx

```diff
@@ -1,95 +1,52 @@
-import Image from "next/image";
-import styles from "./page.module.css";
+"use client";
+
+import { useStepsContext } from "./context/stepsContext";
+import { useState, useEffect } from "react";
+import { useDataContext } from "./context/dataContext";
+import { AppointmentList } from "./components/stepOne/AppointmentList";
+import style from './page.module.css'
+import { PatientDataProvider, DataType } from "./context/patientContext";
+import AppointmentType from "./appointmentType";
 
 export default function Home() {
-  return (
-    <div className={styles.page}>
-      <main className={styles.main}>
-        <Image
-          className={styles.logo}
-          src="https://nextjs.org/icons/next.svg"
-          alt="Next.js logo"
-          width={180}
-          height={38}
-          priority
-        />
-        <ol>
-          <li>
-            Get started by editing <code>src/app/page.tsx</code>.
-          </li>
-          <li>Save and see your changes instantly.</li>
-        </ol>
+  const value = useStepsContext()
+  const info = useDataContext()
+  const [patientData, setPatientData] = useState<DataType>(null)
+  const patientDataValue = { patientData, setPatientData }
+  const newEndTime = new Date()
+  newEndTime.setFullYear(newEndTime.getFullYear() + 1)
+  const [startDate, setStartDate] = useState({ startTime: new Date(), endTime: newEndTime })
```

Comment:
I get the impression you don't need endTime to be in state here, as it never changes after init - just keep as newEndTime and we can simplify by having a startDate as a Date rather than object